src/MatrixClient.ts

```diff
@@ -1715,6 +1715,19 @@ export class MatrixClient extends EventEmitter {
         });
     }
 
+    /**
+     * Upgrade a room. This will call the room upgrade endpoint on the homeserver, which will create a new room with some of the
+     * state carried over. See the spec for which state will be carried over.
+     * @param roomId The room to upgrade
+     * @param newVersion The room version of the new room.
+     * @see https://spec.matrix.org/v1.15/client-server-api/#server-behaviour-19
```

Comment:
Apparently [the URL should be wrapped in `@link`](https://tsdoc.org/pages/tags/see/):
> Note: JSDoc attempts to automatically hyperlink the text immediately after `@see`. Because this is ambiguous with plain text, TSDoc instead requires an explicit {`@link`} tag to make hyperlinks.